### PR TITLE
HQ threat decay description

### PR DESCRIPTION
**Pull request by dcdenu4**
Adding clarity about how threats are decayed when using exponential decay method. This will better line up with InVEST HQ changes that will be released in 3.13.0 where we switch using convolutions to decay threats and use euclidean distance transforms.

Also adding in a new updated figure to demonstrate exponential decay.

@lmandle, @newtpatrol, @jade-md I'm adding you all as reviewers, but it is kind of a pain to review these updates without being able to see the RST compiled nicely. So, feel free to review here OR I can share a built version for viewing and we can edit in a google doc or something.

### CODE REVIEW

**Review by phargogh — APPROVED** [review]

File: source/habitat_quality.rst
Comment (lmandle) [inline source/habitat_quality.rst:74]: This looks great! I only see two v. minor typos: there should be an apostrophe such that the sentence that starts "The threats impact..." should read "The threat's impact..." and then later in that sentence a missing work "to" "set **to** 0 in the model." I'm sorry I'm not sure how to edit directly, otherwise I would have!

**Review by lmandle — CHANGES_REQUESTED** [review]
It's so great to have this issue resolved in the model and fully documented in the UG! I noted some small typos to correct. For the figure "HQ-exponential-decay.png" I can't tell what the caption is, but since we note that irxy at max threat distance = 0.05 and then at distances great than max threat distance, the value is 0, it would be great to show that in the figure as well, such that the lines for each max distance of threat stop or go to zero beyond that distance.

File: source/habitat_quality.rst
Comment (dcdenu4) [reply source/habitat_quality.rst:74]: Thanks Lisa, this is totally fine! There is a way to "suggest a change" by selecting the "+" on a line to comment and then clicking on the "+/-" page looking icon to the right by the other formatting options. Thanks for engaging here!

Comment (dcdenu4) [inline source/habitat_quality.rst:74]:
```suggestion
where :math:`d_{xy}` is the linear distance between grid cells :math:`x` and :math:`y` and :math:`d_{r\ \mathrm{max}}` is the maximum effective distance of threat :math:`r`'s reach across space. For exponential decay, the constant of 2.99 reduces the impact of the threat by 95% (to 5%) at the specified maximum effective distance. This is based on the traditional 95% cutoff used in statistics. The threat's impact beyond the maximum effective distance is set to 0 by the model. Figure 1 illustrates the exponential relationship between the distance-decay rate for a threat based on the maximum effective distance of the threat. For example, if the user selects an exponential decline and the maximum impact distance of a threat is set at 1 km, the impact of the threat on a grid cell's habitat will decline by ~ 55% when the grid cell is 200 m from :math:`r`'s source. If :math:`i_{rxy} > 0` then grid cell :math:`x` is in degradation source :math:`ry`'s disturbance zone. (If the exponential function is used to describe the impact of degradation source :math:`r` on the landscape then the model ignores values of :math:`i_{rxy}` that are very close to 0 in order to expedite the modeling process.) To reiterate, if we have assigned species group-specific habitat suitability scores to each LULC then threat impact over space should be specific to the modeled species group.
```

**Comment by dcdenu4** [comment]
Hey @lmandle, I made those edits you suggested. I also updated the figure to show that it drives to 0 at max distance and added more points for a continuous look.

For the Figure caption, I've left it unchanged with:
>Figure 1. An example of the relationship between the distance-decay rate of a threat and the maximum effective distance of a threat.

So, if that figure caption seems okay and the graph looks good, then you can "Approve" and if you'd like, you can Merge by clicking "Merge pull request" -> "Confirm merge".

In case you're curious, at the top this PR shows we're committing into `natcap:release/3.13.0`. Normally we'd merge right into the `main` branch, but since these changes won't be "live" until the `3.13.0` InVEST release we create a Users Guide release branch to "store" them under. Otherwise the changes would show up in the UG now and be out of sink with the current InVEST release.

I also wanted to note, in case you didn't know, that when reviewing RST files you can select to view them as rendered by clicking the little page icon (highlighted in yellow in the example below)
image

**Comment by newtpatrol** [comment]
I just took a look at Figure 1, and wonder if I might request some improvements to it. Perhaps it will be visually larger in the final release, but as it is, I can barely read any of the text. It would also be useful to have a legend title for what the 1km...5km lines mean. It's also kinda hard to tell some of those color lightness gradations apart, so maybe better to use different colors, and/or label each one somehow (or some other way of differentiating). If all of this makes you say "how about you do that yourself?" I'd be happy to take a shot if you provide whatever source you're working with.

**Review by lmandle — APPROVED** [review]
Looks great!

**Comment by dcdenu4** [comment]
@newtpatrol, since you've kindly offered 😉 , here is the source for the figure! This source will also be committed into the repository, so after you've edited and made a new figure, just upload it back here and I can upload the new figure and source to this PR. I've been working from an Excel file, for reasons, but hopefully this isn't too much a pain to edit... I'm no Excel graph pro, so I was making custom data entries and such to get what I wanted... Good luck!

[HQ-decay-functions-chart.xlsx](https://github.com/natcap/invest.users-guide/files/10711062/HQ-decay-functions-chart.xlsx)

**Comment by dcdenu4** [comment]
Okay, I chatted with Stacie and we are going to go with the updated figure I made. We can always change it at any time.

**Pull request merged**
